boa/src/syntax/parser/statement/iteration/do_while_statement.rs

```diff
@@ -0,0 +1,73 @@
+use crate::syntax::{
```

Comment:
Missing documentation.
```suggestion
//! Do-while statement parsing.
//!
//! More information:
//!  - [MDN documentation][mdn]
//!  - [ECMAScript specification][spec]
//!
//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while
//! [spec]: https://tc39.es/ecma262/#sec-do-while-statement

use crate::syntax::{
```